user_manual/talk/advanced_features.rst

```diff
@@ -296,4 +291,14 @@ You can generate it by pressing the button that is visible in chat above the fir
    :width: 500px
 
 .. image:: images/chat-summary-text.png
+   :width: 500px
+
+Call live transcription
+-----------------------
+
+Call live transcription allows to transcribe the speech in real-time during a call. It is set up by the system administration (high-performance backend and `Live Transcription App <https://apps.nextcloud.com/apps/live_transcription>`_ are required).
```

Comment:
```suggestion
Call live transcription allows to transcribe the speech in real-time during a call. It can be set up by the system administration (High-performance backend and `Live Transcription App <https://apps.nextcloud.com/apps/live_transcription>`_ are required).
```